ng2-components/ng2-alfresco-documentlist/README.md

```diff
@@ -357,6 +360,29 @@ DocumentList provides simple breadcrumb element to indicate the current position
 | --- | --- | --- |
 | navigate | [PathElementEntity](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-core-rest-api/docs/PathElementEntity.md) |emitted when user clicks on a breadcrumb  |
 
+## Dropdown Site Component
+
+DocumentList provides simple dropdown element to indicate the current position within a navigation hierarchy.
+
+```html
+ <adf-sites-dropdown  
+  [showDefaultOption]="true"
+  (siteChanged)="getSiteContent($event)">
+ </adf-sites-dropdown>
+```
+
+### Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| showDefaultOption | boolean | (optional) Shows into the dropdown a default option |
```

Comment:
To enable/disable from outside the default option in a select. Sometimes you want a default one which has no value associated.